demisto_sdk/commands/format/tests/test_formatting_yml_test.py

```diff
@@ -646,14 +646,53 @@ def test_update_tests_on_integration_with_test_playbook(self):
             - Ensure `tests` field gets the Test Playbook ID
         """
         test_files_path = os.path.join(git_path(), 'demisto_sdk', 'tests')
-        vmware_integration_yml_path = os.path.join(test_files_path, 'test_files', 'content_repo_example', 'Packs',
-                                                   'VMware',
+        vmware_integration_yml_path = os.path.join(test_files_path, 'test_files', 'Packs',
+                                                   'Phishing',
                                                    'Integrations', 'integration-VMware.yml')
         formatter = IntegrationYMLFormat(input=vmware_integration_yml_path, output='')
         res = formatter.update_tests()
         assert res is None
         assert formatter.data.get('tests') == ['VMWare Test']
 
+    @patch('builtins.input', lambda *args: 'no')
+    def test_update_tests_on_playbook_with_test_playbook(self):
```

Comment:
Is this test a reproduce of the bug with the expected result?

Reply:
yes, checkout my comment on this pr